django/test/runner.py

```diff
@@ -477,6 +481,9 @@ def __init__(self, subsuites, processes, failfast=False, buffer=False):
         self.serialized_contents = None
         super().__init__()
 
+    def process_setup(self, *args):
+        pass
+
```

Comment:
Does this merit a comment at all? 🤔

Reply:
I added a comment.

Reply:
Thanks. I think that gives a pointer in the right direction. 👍